src/scripts/seedDb.js

```diff
@@ -179,7 +179,7 @@ async function createDefaultMLModels({ defaultMLModelsConfig }) {
 }
 
 async function createDefaultProjects({ defaultProjectsConfig }) {
-  console.log('Creaing default projects...');
+  console.log('Creating default projects...');
   let existingProjIds = await await Project.find({}, '_id');
```

Comment:
Looks like a typo with 2 `await` statements?

Reply:
Yeah looks like it's a typo to me too. Mind fixing it?

Reply:
The whole seedDB script is probably out of date and needs updating now that we have `Project.labels`.